diagnostic.md

```diff
@@ -34,5 +37,7 @@ Record your responses inside the fenced code blocks below each question.
     pass it data?
 
     ```html
-    <!-- your response here -->
+    {{#each model.phones as |phone|}}
+      {{my-phone phone=phone}}
+    {{/each}}
```

Comment:
👍 

Because this is a component, it's unlikely that the data would be passed down as `model`. More likely it would be `contact.phones`.